IdokladSdk/Requests/IssuedTaxDocument/Filter/IssuedTaxDocumentFilter.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using IdokladSdk.Enums;
+using IdokladSdk.Models.IssuedTaxDocument.Get;
+using IdokladSdk.Models.IssuedTaxDocument.List;
+using IdokladSdk.Requests.Core.Modifiers.Filters;
+using IdokladSdk.Requests.Core.Modifiers.Filters.Common;
+
+namespace IdokladSdk.Requests.IssuedTaxDocument.Filter
+{
+    /// <summary>
+    /// Filterable properties for <see cref="IssuedTaxDocumentGetModel"/>.
+    /// </summary>
+    public class IssuedTaxDocumentFilter : IdFilter
+    {
+        /// <inheritdoc cref="IssuedTaxDocumentListGetModel.AccountedByInvoiceId"/>
+        public FilterItem<int> AccountedByInvoiceId { get; set; } = new FilterItem<int>(nameof(IssuedTaxDocumentGetModel.AccountedByInvoiceId));
+
+        /// <inheritdoc cref="IssuedTaxDocumentListGetModel.CurrencyId"/>
+        public FilterItem<int> CurrencyId { get; set; } = new FilterItem<int>(nameof(IssuedTaxDocumentGetModel.CurrencyId));
+
+        /// <inheritdoc cref="IssuedTaxDocumentListGetModel.DateOfIssue"/>
+        public CompareFilterItem<DateTime> DateOfIssue { get; set; } = new CompareFilterItem<DateTime>(nameof(IssuedTaxDocumentGetModel.DateOfIssue));
+
+        /// <inheritdoc cref="IssuedTaxDocumentListGetModel.DateOfTaxing"/>
+        public CompareFilterItem<DateTime> DateOfTaxing { get; set; } = new CompareFilterItem<DateTime>(nameof(IssuedTaxDocumentGetModel.DateOfTaxing));
+
+        /// <inheritdoc cref="IssuedTaxDocumentListGetModel.DocumentNumber"/>
+        public CompareFilterItem<string> DocumentNumber { get; set; } = new CompareFilterItem<string>(nameof(IssuedTaxDocumentGetModel.DocumentNumber));
```

Comment:
Opět bych změnil typ property na ContainFilterItem - u DocumentNumber to tak v ostatních entitách máme.